tripy/tripy/frontend/ops/repeat.py

```diff
@@ -95,6 +95,6 @@ def repeat(input: "tripy.Tensor", repeats: Union[int, "tripy.ShapeScalar"], dim:
     repeat_mask = concatenate(
         [reshape(repeats, (1,)) if idx == dim else Tensor([1]) for idx in range(input.rank)], dim=0
```

Comment:
Does changing this part to:
```suggestion
        [reshape(repeats, (1,)) if idx == dim else Shape([1]) for idx in range(input.rank)], dim=0
```
remove the need to cast the `repeat_mask` to a `Shape` below?

Reply:
Actually, now I'm wondering if it can be simplified to:
```py
repeat_mask = [1] * input.rank
repeat_mask[dim] = repeats
```
That seems more natural to write (and probably what a user would expect to be able to do).